example-recording-scheduler.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=CRS runner for script-A-recording-scheduler.pl
+After=network.target
+
+[Service]
+User=voc
+Group=voc
+ExecStart=/opt/crs-scripts/bin/crs_run script-A-recording-scheduler.pl
+WorkingDirectory=/opt/crs-scripts
+Restart=on-failure
+RestartSec=10
+SyslogIdentifier=crs-recording-scheduler
```

Comment:
Wouldnt it make sense to add an EnvironmentFile here, that contains CRS_SECRET etc.?

Reply:
Yes, no, not sure tbh.

I wanted to include a basic file which preserves as much as the former usability of `bin/crs_run`, so we still have the feature of allowing multiple environment file locations and such.

I will probably create a follow-up PR containing more changes to this stuff, unless you want me to include all of that in here.

Reply:
Oh sorry, I did not see that the unit executes crs_run - but makes sense since the perl scripts usually wont start on their own.

Another full blown example would be nice. As far as I can see, all features can be done with unit files: multiple environment files, setting PERL5LIBS variable and different restart-delays depending on the exit code. thats about all there is in crs_run I think

Reply:
I haven't found any documentation on having different restart-delays based on exit code in systemd, can you point me to that?
I will implement the remaining stuff in the systemd unit, if that's what we want.

Reply:
You are right, I thought there was an option related to SuccessExitStatus and RestartSteps, but there is nothing that is dependent on the exit of the command. So maybe the sleep should be included in the perl scripts. It is just an optimization to not hammer the tracker but give fast response. In doubt, everything will work just fine with a fixed restart interval of 30 seconds. Some scripts, like script A, are already consuming the tickets in a loop, which makes the sleep time less important. It might be worth the effort to include a polling loop in the lib/CRS and then exit the script after x polling attempts or if a ticket has been processed. Then the systemd restart delay is not important.

However, Restart should be set to "always".

Reply:
Yeah, we could start refactoring the scripts - but then, maybe we want to re-write them in a language more voc people understand (probably python?).

Anyways, i think refactoring or rewriting the scripts is outside the scope of this PR.

Reply:
yes, indeed. And if you want to rewrite them because you think perl is a problem - go ahead and do it.

Reply:
Well, *I* like perl :D